src/components/Nav/RouterComponent.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Redirect, Route, Switch } from 'react-router';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Welcome } from '../../pages';
+import Nav from './Nav';
+
+const RouterComponent = (props) => {
+  const { token, list, loading, error, updateToken } = props;
+  return (
+    <Router>
+      <Switch>
+        <Route
+          exact
+          path="/:page"
+          render={(props) => (
+            <Nav
+              {...props}
+              token={token}
+              list={list}
+              loading={loading}
+              error={error}
+              userToken={token}
+              updateToken={updateToken}
+            />
+          )}
+        />
+        <Route exact path="/">
+          {props.token ? (
+            <Redirect to="/list" />
+          ) : (
+            <Welcome updateToken={props.updateToken} />
+          )}
```

Comment:
This is not a high priority, but if I create a list and start adding items to it, there is no way for me to go back to the home page and either join another or create a new list. I would have to manually go into my local storage and the delete the token. It would be nice if when I have a token, I can still navigate to the home page to from the bottom navigation. Would you be able to remove this conditional and add a navigation tab for the home page?

```suggestion
          <Welcome updateToken={props.updateToken} />
```